static/app/views/preprod/types.ts

```diff
@@ -1,148 +1,40 @@
-/**
- * TypeScript types for size analysis treemap visualization
- */
+export type Platform = 'ios' | 'android';
 
-export enum TreemapType {
-  // Generic file categories (cross-platform)
-  FILES = 'files',
-  EXECUTABLES = 'executables',
-  RESOURCES = 'resources',
-  ASSETS = 'assets',
-  MANIFESTS = 'manifests',
-  SIGNATURES = 'signatures',
-  FONTS = 'fonts',
-
-  // iOS-specific categories
-  FRAMEWORKS = 'frameworks',
-  PLISTS = 'plists',
-
-  // Android-specific categories
-  DEX = 'dex',
-  NATIVE_LIBRARIES = 'native_libraries',
-  COMPILED_RESOURCES = 'compiled_resources',
-
-  // Binary analysis categories (cross-platform)
-  MODULES = 'modules',
-  CLASSES = 'classes',
-  METHODS = 'methods',
-  STRINGS = 'strings',
-  SYMBOLS = 'symbols',
-
-  // iOS binary categories
-  DYLD = 'dyld',
-  MACHO = 'macho',
-  FUNCTION_STARTS = 'function_starts',
-  CODE_SIGNATURE = 'code_signature',
-  EXTERNAL_METHODS = 'external_methods',
-
-  // Generic categories
-  OTHER = 'other',
-  UNMAPPED = 'unmapped',
+export interface BuildDetails {
+  app_info: BuildDetailsAppInfo;
+  state: BuildDetailsState;
+  vcs_info: BuildDetailsVcsInfo;
 }
 
-export interface TreemapElement {
-  /** Child elements */
-  children: TreemapElement[];
-  /** Platform and context-specific metadata */
-  details: Record<string, unknown>;
-  /** Download size in bytes (compressed) */
-  download_size: number;
-  /** Install size in bytes */
-  install_size: number;
-  /** Whether this element represents a directory */
-  is_directory: boolean;
-  /** Display name of the element */
+export interface BuildDetailsAppInfo {
```

Comment:
Unsure if there's a better way to define BE<>FE models. Will ask around tomorrow/next week